### PR TITLE
Add docs on Cookie length limitations

### PR DESCRIPTION
**Pull request by BrandonRomano**
> [!IMPORTANT]
> 🔎 Previews:
> - https://clerk.com/docs/pr/1700/authentication/configuration/session-options

Includes details on browser limitations / user behaviors which may cause a Clerk session cookie to be deleted, which results in the user being signed out.

### CODE REVIEW

File: docs/authentication/configuration/session-options.mdx [hunk 1]
```diff
@@ -41,7 +44,25 @@ By default, this setting is enabled with a default value of 7 days for all newly
...
+
+Regardless of how [session lifetimes](#session-lifetime) are configured, there are certain browser limitations & behaviors which may clear Clerk's session cookie. This will cause users to be signed out, even if your session lifetimes are set to a longer duration. As a result, it is impossible to achieve a setup where your users are never signed out.
+
+### User behaviors
+
+In the event that a user manually clears their cookies, Clerk's session cookie will be lost. Similarly, if a user signs in via an incognito window and they then close all incognito windows, Clerk's session cookie will be lost. Both of these scenarios will cause the user to have to sign in again.
+
+### Safari
+
+Safari will clear Clerk's session cookie every 7 days as an effect of their [CNAME cloaking ITP policy](https://webkit.org/blog/11338/cname-cloaking-and-bounce-tracking-defense/). This will cause users to be signed out weekly. The only current workaround to this issue is to [proxy FAPI](/docs/advanced-usage/using-proxies).
```
Comment (jescalan) [inline docs/authentication/configuration/session-options.mdx:59]: Oh this is actually wrong now - due to some fine handywork from @issuedat, I think we can entirely remove this section

File: docs/authentication/configuration/session-options.mdx [hunk 2]
```diff
@@ -41,7 +44,25 @@ By default, this setting is enabled with a default value of 7 days for all newly
...
+
+### Google Chrome
+
+Cookies set in Google Chrome have a Max-Age upper limit of [400 days](https://developer.chrome.com/blog/cookie-max-age-expires). Users who are using Google Chrome will be signed out within 400 days, even if session lifetime is set to a longer duration. There is no workaround for this.
```
Comment (jescalan) [inline docs/authentication/configuration/session-options.mdx:63]: Honestly I wonder if we should have a note about this in the dashboard, or at least a little info thing that points to this doc.

**Review by jescalan — COMMENTED** [review]
This is great, thank you!

File: docs/authentication/configuration/session-options.mdx (hunk 1 above)
Comment (BrandonRomano) [reply docs/authentication/configuration/session-options.mdx:59]: Oh, amazing!  I will remove this.

File: docs/authentication/configuration/session-options.mdx (hunk 2 above)
Comment (BrandonRomano) [reply docs/authentication/configuration/session-options.mdx:59]: I think I'm going to make it so a warning pops up when the user configures > 400 days (as compared to the basic UI i have right now)

Comment (jescalan) [reply docs/authentication/configuration/session-options.mdx]: I like this

**Review by jescalan — COMMENTED** [review]
✅

**Pull request merged**